src/backend/controllers/crowdsourcer/__tests__/crowdsourcer.test.ts

```
@@ -70,8 +70,8 @@ describe('crowdsourcer', () => {
 
       // assert
       expect(mockResponse.status).toHaveBeenCalledWith(403);
-      expect(sendMock).toHaveBeenCalledWith({
-        error: `Users cannot be referred twice. Referral code [${referralCode}] will be ignored`,
+      expect(sendMock).toHaveBeenNthCalledWith(1, {
```

why change to `toHaveBeenNthCalledWith`?

---

actually my bad! i need to update the controller to return early (i personally don't like doing) so send is only called once if its a 403. Ideally we should throw an error but i dont think express will intercept it and relay the correct message to the client